site/en/dev-best-practice.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: page
+title: How we want to develop software in OCR-D
+lang: en
+lang-ref: best-practices
+---
+
+# Best Practices for Software Development in OCR-D
+
+This guide is informed by the experience of coordinating distributed development of the [OCR-D software and specifications](https://github.com/OCR-D). We strive for a broad consensus on the practicalities and logistics of software development, particular in preparation of the upcoming [Phase III of OCR-D](https://ocr-d.de/en/2020/06/04/pilot.html).
+
+## Communications
+
+> [Be bold](https://en.wikipedia.org/wiki/Wikipedia:Be_bold)
+
+Software Development is as much a social endeavor as it is technical. To effectively develop in a distributed setting, transparency and sharing early in the process is vital. We therefore strongly recommend using the following channels over E-Mail.
+
+### GitHub Account
+
+Since all development shall be openly and transparently carried out on GitHub, it is essential that you [create a GitHub account](https://github.com/join).
+
+### Chat
+
+The [OCR-D gitter "Lobby"](https://gitter.im/OCR-D/Lobby) is the general channel for all things OCR-D. We strive to keep the threshold for participation as low as possible. Every question or is welcome. It is also the channel we use to announce changes to [OCR-D core](https://github.com/OCR-D/core), the [specifications](https://github.com/OCR-D/spec) or updates on [processors](https://github.com/topics/ocr-d).
+
+### GitHub Issues
+
+As with the [chat](#chat), we aim for low barriers of entry for participation on issues in GitHub. If something is not working within OCR-D projects, you encounter error messages or unexpected behavior: Feel free to open an issue. If you cannot identify which component or project is the cause of the problem, [open an issue in OCR-D/core](https://github.com/OCR-D/core/issues/new), the moderators can transfer it to the right place later.
+
+This openness must go both ways: If you maintain or contribute to a project, it is essential that we treat each other with respect. Many users and even developers are still learning about OCR, software development and distributed version control. A clueless "newbie" can become a helpful contributor if you support them.
+
+### GitHub Forks
+
+A *fork* is a clone of a GitHub repository hosted on the site. If you fork a repository, a clone of the repository is created under your username. The advantage of a fork over just the original repository is that you have full access to it and can make changes to it to fit your use case. If those changes seem useful to others, you can create a [pull request](#pull-request)
```

Comment:
```suggestion
A *fork* is a clone of a GitHub repository hosted on the same site. If you fork a repository, a clone of it will be created under your username. The advantage of a fork over just the original repository is that you have full control to it and can make changes to it to fit your use case, while others also can access it. If you deem your changes a useful contribution to the original project, you can create a [pull request](#pull-request) to allow discussing and refining them with others, and ultimately merge.
```

Reply:
full control **of** it